spec/models/game_spec.rb

```diff
@@ -116,4 +116,25 @@
       expect(game.pieces.count).to eq(32)
     end
   end
+
+  describe 'end game' do
```

Comment:
it's good to have the method name in the describe.

Like this

`describe '#forfeit'`

#method_name = "this is an instance method"
.method_name = "this is a class method"